docs/reference/cli.md

```diff
@@ -44,314 +47,433 @@ The line `Telemetry enabled` disappears when you disable it.
 `quickwit --version` displays the version. It is useful for reporting bugs.
 
 
+### Syntax
```

Comment:
It would be nice to introduce early in the doc the fact that we can use the env variable `QW_CONFIG` to set the config once for all.